transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModelTest.java

```diff
@@ -156,14 +159,21 @@ public void testCreateTransferViewModel_initializeWithoutFundingSource() {
         assertThat(viewModel.isTransferAllAvailableFunds(), is(notNullValue()));
         assertThat(viewModel.isTransferAllAvailableFunds().getValue(), is(false));
         assertThat(viewModel.isCreateQuoteLoading().getValue(), is(false));
+        
+        verify(viewModel, never()).loadTransferSource();
     }
 
     @Test
-    public void testCreateTransferViewModel_verifyDefaultValuesWithFundingSource() {
+    public void testInit_withFundingSource() {
         CreateTransferViewModel viewModel = new CreateTransferViewModel.CreateTransferViewModelFactory(
-                "src-token", mTransferRepository, mTransferMethodRepository, mUserRepository
+                mTransfer.getToken(), mTransferRepository, mTransferMethodRepository, mUserRepository
         ).create(CreateTransferViewModel.class);
 
+        CreateTransferViewModel mockedViewModel = spy(viewModel);
+        mockedViewModel.init();
+
+        verify(mockedViewModel, times(1)).loadTransferDestination(any(String.class));
```

Comment:
verify defaults to execute once so no need to be explicit in this case

Reply:
we can also verify which load is being invoked

Reply:
Done